ngraph/core/src/node.cpp

```diff
@@ -803,7 +803,7 @@ bool ov::Node::evaluate_upper(const HostTensorVector& output_values) const {
 bool ov::Node::constant_fold(OutputVector& output_values, const OutputVector& input_values) {
     OV_ITT_SCOPED_TASK(ov::itt::domains::nGraph, "Node::constant_fold");
 
-    if (m_rt_info.count("DISABLED_CONSTANT_FOLDING")) {
+    if (m_rt_info.count("disabled_constant_folding_0")) {
```

Comment:
can we have a helper like `has` or smth?

Reply:
it is already exists, but unfortunately there is a problem that ngraph target doesn't depend on transformation library where this attribute is located. The only right solution here is to move disable_constant_folding attribute to ngraph library. But I to make this transition in separate PR.